### ISSUE TITLE
xargs: illegal option -- r

### ISSUE THREAD

**Issue by avevlad** [rlivings39/vscode-fzf-quick-open#10]
> Open file using fzf

```bash
fzf --print0 | xargs -0 -r code
xargs: illegal option -- r
                          usage: xargs [-0opt] [-E eofstr] [-I replstr [-R replacements]] [-J replstr]
                                [-L number] [-n number [-x]] [-P maxprocs] [-s size]
              [utility [argument ...]]
```

```
ProductName:	Mac OS X
ProductVersion:	10.15.3
BuildVersion:	19D76
```

**Comment by rlivings39** [comment]
More reason for me to move to Typescript implementation. I'll put in a special case for Mac for now.

**Comment by avevlad** [comment]
@rlivings39 thanks

**Comment by rlivings39** [comment]
Just published v0.2.5. Let me know if that does the trick as I don't have a Mac with fzf to test on.

**Comment by avevlad** [comment]
@rlivings39 work 🎉Thanks!
